src/package.json

```diff
@@ -30,6 +30,7 @@
     "maxmind": "4.3.1",
     "nanoid": "3.1.20",
     "node-pty": "0.9.0",
+    "osx-temperature-sensor": "1.0.7",
```

Comment:
Please don't add osx-temperature-sensor here. It's an optional dep and should stay optional to avoid breaking stuff on some non-osx builds.

```suggestion
```

Reply:
ahh, both of those deps were added due to me running install and it auto saved them for the dependency tree, they should stay optional.